README.md

```diff
@@ -74,6 +74,19 @@ There are manually maintained jobs and some jobs are now using
 [jenkins-job-builder](http://docs.openstack.org/infra/jenkins-job-builder/)
 which defines jobs in yaml format. New jobs should always be defined
 in yaml format.
+
+A fast guide to setup jenkins job:
+
+* Install SUSE CA certificate package as described in http://ca.suse.de/gettingstarted/
+* Install jenkins-job:
+```
+virtualenv .venv
+. .venv/bin/activate
+pip install jenkins-job-builder
```

Comment:
I used a different approach: ` zypper ar devel_languages_python && zypper in python-jenkins-job-builder`
For me and those not familiar with `pip` and `virtualenv` it might be an easier option. Is there is any reason to prefer one over the other?